### PR TITLE
Change Azure active directory lookup to Auto.....

### PR DESCRIPTION
**Pull request by ZhehooMorgen**
Setting name changed, will be effactive soon

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/SPMT-settings.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The following table details the settings available in SPMT.  Advanced settings s
 
 |**Setting**|**Description**|
 |:-----|:-----|
-|Azure Active Directory lookup |By default, this is set to **On**. If no user mapping file is provided by the user, then Azure Active Directory is used as the default for user mapping.|
+|Automatic user mapping|By default, this is set to **On**. If no user mapping file is provided by the user, then Azure Active Directory is used as the default for user mapping on shareponit 2013 and mapping based on user name will be used on sharepoint 2010.|
```
Comment (JohanFreelancer9) [inline migration/SPMT-settings.md]:
```suggestion
|Automatic user mapping|By default, this is set to **On**. If no user mapping file is provided by the user, then Azure Active Directory is used as the default for user mapping on SharePoint 2013 and mapping based on user name is used on SharePoint 2010.|
```

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**